src/d/a/d_a_base.cpp

```diff
@@ -463,8 +527,38 @@ SoundSource *dAcBase_c::getSoundSource() {
 // End of SoundSource stuff
 
 // first param is not dAcBase_c
+// - are we sure? 8002c1a0 seems to suggest otherwise but this func does behave very weird
+// NOT MATCHING
```

Comment:
These are linked list inlines, dowsing_target has similar code. The first part is weird in dowsing_target too, the rest is a normal remove inline